materialize/resources/resource_secret.go

```diff
@@ -40,7 +44,7 @@ func Secret() *schema.Resource {
 			"value": {
 				Description: "The value for the secret. The value expression may not reference any relations, and must be implicitly castable to bytea.",
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
```

Comment:
Needed to make value optional, otherwise you can't import an existing secret without setting it which may not be known